payid-stack.yaml

```diff
@@ -458,6 +458,30 @@ Resources:
             DNSName: !GetAtt "apiGatewayCustomDomain.DistributionDomainName"
             HostedZoneId: !GetAtt "apiGatewayCustomDomain.DistributionHostedZoneId"
 
+  cloudwatchLogsLogGroupListCertificatesFn:
```

Comment:
Question for you: did these work for you? I tried to add something similar on my first go at this and ended up having it not work as expected.

For what it's worth the log groups should end up being created dynamically via this part of the policy that's present in each of the functions:
```
Policies:
        - PolicyDocument:
            Version: "2012-10-17"
            Statement:
              - Action:
                  - "logs:CreateLogGroup"
                  - "logs:CreateLogStream"
                  - "logs:PutLogEvents"
                Effect: "Allow"
                Resource: "*"
          PolicyName: "create-test-account-logs"
        - PolicyDocument:
            Version: "2012-10-17"
            Statement:
              - Action:
                  - "s3:*"
                Effect: "Allow"
                Resource:
                  - "*"
          PolicyName: "create-test-account-s3"
```

AWS automatically creates the group (though it didn't automatically create the retention policy). 

Just wanted to ask before merging since this tripped me up a while back.

Reply:
They did work for me but I was doing a full-stack creation. Were you trying to do an update to an existing stack maybe? If that is the case I can see it failing. The reason why I added them was for the retention policy, if you don't see the need for them I will understand.

Reply:
Huh, no, I was creating from scratch. I must have botched something when I first wrote them. I'll leave them in there then; seems like you solved two problems for me 👍